css/style.css

```diff
@@ -22,7 +28,7 @@ body {
   background: #fff;
 }
 
-.navbar-brand {
+.navbar-light .navbar-brand, .navbar-light .navbar-brand:hover, .navbar-light .navbar-brand:visited {
```

Comment:
CSS selectors should be on separate lines. Fix this manually for now, we'll add linting for styles soon.